src/domain/banner/bannerHero/BannerHero.tsx

```diff
@@ -100,7 +101,8 @@ const BannerHero: React.FC<Props> = ({ banner, location }) => {
   const testIds = getTestIds(location);
 
   return (
-    <div
+    <VisibleAfterComponentMounted
```

Comment:
Are we going to need these to get around ssr problems in HDS or?

Reply:
In a way yes, without this the banner image and text will jump around before their final position (after client side render). The idea is to hide the component until it is mounted. With "Visiblity: hidden" the component still would reserve the same space even when not visible.